pyscriptjs/src/styles/pyscript_base.css

```diff
@@ -330,3 +330,20 @@ textarea {
 .py-terminal-hidden {
     display: none;
 }
+
+/* avoid changing the page layout when the terminal is docked and hidden */
+html:has(py-terminal[docked]:not(py-terminal[docked].py-terminal-hidden)) {
+    padding-bottom: 40vh;
+}
+
+py-terminal[docked] {
+    position: fixed;
+    bottom: 0;
+    width: 100vw;
+    max-height: 40vh;
```

Comment:
this means "40% of the viewport height"?
I'm happy with this value for now, but we probably need to think more about that we want for small screens and mobile phones.

Reply:
that's why is percentage ... if having a reading area bigger than writing area for smaller screens makes sense, I can use media query to target exactly those small screens. I don't have any strong opinion here because admittedly I haven't used this REPL much to date and surely never on mobile, so whatever you think is better as percentage, works for me.

Reply:
I think 40% is fine for now. I was mostly thinking aloud but wanted to raise the concern to make sure that we think about it in the future